user_sessions/views.py

```diff
@@ -43,3 +43,21 @@ class LogoutView(View):
     def get(self, request):
         logout(request)
         return HttpResponseRedirect(reverse('user_sessions:home'))
+
+
+class AddUserView(View):
+
+    form_class = UserRegistrationForm
+    template_name = 'user_sessions/register_user.html'
+
+    def get(self, request):
+        form = self.form_class()
+        return render(request, self.template_name, {'form': form})
+
+    def post(self, request):
+        form = self.form_class(request.POST)
+        if form.is_valid():
+            form.save()
```

Comment:
you were doing it right earlier! user shouldn't be asked to login after signup, instead we should logged him automatically and land him on dashboard page.
lets authenticate user n then login right here.